packages/supabase/src/liveProvider/index.ts

```diff
@@ -49,19 +66,37 @@ export const liveProvider = (
         }
       };
 
-      const mapFilter = (filters?: CrudFilters): string | undefined => {
+      const mapFilter = (
+        filters?: CrudFilters,
+        meta?: any,
+      ): string | undefined => {
         if (!filters || filters?.length === 0) {
           return;
         }
 
-        return filters
+        if (filters.length > 1 && !meta?.realtimeFilter) {
+          warnOnce(
+            true,
+            `Warning: Multiple filters detected for resource "${resource}". Supabase Realtime currently supports only a single filter. The first filter will be applied. To customize this behavior, use the 'meta.realtimeFilter' property.`,
+          );
+        }
```

Comment:
```suggestion
        warnOnce(
          filters.length > 1 && !meta?.realtimeFilter,
          `Warning: Multiple filters detected for resource "${resource}". Supabase Realtime currently supports only a single filter. The first filter will be applied. To customize this behavior, use the 'meta.realtimeFilter' property.`,
        );
```

I think this looks more correct use for warnOnce 🤔